packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/TelemetryView.tsx

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+
+import {
+	handleIncomingMessage,
+	IDebuggerMessage,
+	InboundHandlers,
+	TelemetryEventMessage,
+} from "@fluid-tools/client-debugger";
+import { ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+import { _TelemetryView } from "@fluid-tools/client-debugger-view";
+import { MessageRelayContext } from "./MessageRelayContext";
+
+const loggingContext = "EXTENSION(DebuggerPanel:Telemetry)";
+
+/**
+ * Displays telemetry events generated by FluidFramework in the application.
+ *
+ * @remarks Must be run under a {@link MessageRelayContext}.
+ */
+export function TelemetryView(): React.ReactElement {
+	const messageRelay = React.useContext(MessageRelayContext);
+	if (messageRelay === undefined) {
+		throw new Error(
```

Comment:
What might happen if meesageRelayConext loading takes long time, will it kills the whole render wondow process? Should we provide a retry?

Reply:
I'll defer to @Josmithr a bit here, he probably understands this better than I do, but I don't think the React context requires any loading time. The `MessageRelayContext' class has no initialization logic so there should be no delay there. As long as we don't make it complex ourselves, I think this should be fine.